core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1523,7 +1523,8 @@ public void run() {
 
     @Override
     public ManagedChannelBuilder<?> createResolvingOobChannelBuilder(String target) {
-      return createResolvingOobChannelBuilder(target, new DefaultChannelCreds());
+      return createResolvingOobChannelBuilder(target, new DefaultChannelCreds())
+          .overrideAuthority(getAuthority());
```

Comment:
This is a subtle difference between the two methods. It should be documented.

Reply:
Added comment.

Reply:
Where? Your comment says "use the other one," not "this method is slightly different than the other one because it X"

Reply:
X is the comment `// Override authority to keep the old behavior` above.

Reply:
Oh, no. I was meaning in the normal javadoc for the method that callers are supposed to see.